components/script/dom/htmlcollection.rs

```diff
@@ -113,37 +112,48 @@ impl HTMLCollection {
         }
     }
 
-    pub fn by_tag_name(window: &Window, root: &Node, mut tag: DOMString)
```

Comment:
I didn't see much value in having separate `by_tag_name` and `by_atomic_tag_name` functions, so I just merged them into one.